EpsilonCCS/Src/CanParsers/CanParser.c

```diff
@@ -80,11 +87,14 @@ void HAL_CAN_RxCpltCallback(CAN_HandleTypeDef* hcan)
     msg->StdId = hcan->pRxMsg->StdId;
 
     memcpy(msg->Data, hcan->pRxMsg->Data, 8);
+    osMessagePut(canRxQueueId, (uint32_t)msg, 0);
 
-    if (osMessagePut(canRxQueueId, (uint32_t)msg, 0) == osOK)
+    if (HAL_CAN_Receive_IT(hcan, CAN_FIFO0) == osOK)
```

Comment:
Why isn't the queue being used? Doesn't this have race condition issues with just straight using the handler?

Reply:
Where's still being used (line 90), but I wanted the LEDs to tell us about the recieved function.